src/components/settings/GeneralSettings.vue

```diff
@@ -38,6 +38,35 @@
 
       <v-divider></v-divider>
 
+      <app-setting :title="$t('app.setting.label.formatdatetime')">
+        <v-select
+          filled
+          dense
+          hide-details="auto"
+          :items="[
+            { text: $t('app.setting.dateformat.monthdayyear'), value: 'MMM. DD' },
+            { text: $t('app.setting.dateformat.daymonthyear'), value: 'DD MMM.' }
+          ]"
+          item-value="value"
+          item-text="text"
+          v-model="dateformat">
+        </v-select>
```

Comment:
A small space between the two options would be nice (either a small margin or just adding an extra `&nbsp;` between them)